docs: add metadata to all remaining pages (#1246)
canonical/rockcraft: docs/how-to/crafting/include-local-files-and-remote-resources.rst

.. meta::

>     :description: How to include local files and remote resources in a Rockcraft build. Learn to use the dump plugin and filesets to add content from your filesystem or remote URLs.

Reviewer: ```suggestion
    :description: How to add arbitrary files to a rock's filesystem. Use the Dump plugin and filesets to add files from local storage or remote URLs.
```
